verilog/analysis/checkers/struct_union_name_style_rule_test.cc

```diff
@@ -44,6 +53,41 @@ TEST(StructUnionNameStyleRuleTest, ValidStructNames) {
   RunLintTestCases<VerilogAnalyzer, StructUnionNameStyleRule>(kTestCases);
 }
 
+TEST(StructUnionNameStyleRuleTestConfigured, ValidStructNames) {
+  const absl::string_view exceptions = "exceptions:12B,11GB,14kJ,B10,t";
```

Comment:
I think the exceptions in the original issue should be about the Unit names only. I.e. this test case needs to be: "exceptions:B,GB,kJ"

Reply:
I think with just using the units and not numbers, your last concern would go away @pawelsag ?

Reply:
Yes, thanks for this. I am about to update the PR